### PR TITLE
fix phone format function

### PR DESCRIPTION
**Pull request by IbrahimAldereni-socuimtech**
- fix phone format function to accept any number length

### CODE REVIEW

**Review by mohdabed123 — APPROVED** [review]

File: src/utils/helpers.js [hunk 1]
```diff
@@ -43,8 +43,13 @@ export const toUpperCaseFirstLetter = (string) =>
  * @return {String} format phone number
```
Comment (AhmadAlShwaiki) [inline src/utils/helpers.js:43]: Please add a descriptive comment example:
Format phone number to become 000-0000-00

File: src/utils/helpers.js [hunk 2]
```diff
@@ -43,8 +43,13 @@ export const toUpperCaseFirstLetter = (string) =>
...
  */
 export const formatPhoneNumber = (phoneNumberString) => {
+  const lastPartLength = phoneNumberString.length - 6;
+  const reg = `^(\\d{3})(\\d{3})(\\d{${lastPartLength}})$`;
+  const regex = new RegExp(reg);
```
Comment (AhmadAlShwaiki) [inline src/utils/helpers.js:48]:
```suggestion
  const regex = new RegExp(`^(\\d{3})(\\d{3})(\\d{${lastPartLength}})$`);
```

File: src/utils/helpers.js [hunk 3]
```diff
@@ -43,8 +43,13 @@ export const toUpperCaseFirstLetter = (string) =>
...
+
   const cleaned = ` ${phoneNumberString}`.replace(/\D/g, '');
-  const match = cleaned.match(/^(\d{3})(\d{3})(\d{3})$/);
+  const match = cleaned.match(regex);
+
   if (match) {
     return `(${match[1]})  ${match[2]} - ${match[3]} `;
```
Comment (AhmadAlShwaiki) [inline src/utils/helpers.js:54]: Change to descriptive naming please, what does match[1] mean?
example: 
```suggestion
    return `(${CountryCode})  ${AreaCode} - ${LineNumber} `;
```

File: src/utils/helpers.js [hunk 4]
```diff
@@ -38,16 +38,42 @@ export const toUpperCaseFirstLetter = (string) =>
...
+  const match = cleaned.match(regex);
+
+  const countryCode = match[1];
+  const areaCode = match[2];
+  const lineNumber = match[3];
+
+  if (match && phoneNumberString?.length > 6) {
+    return `(${countryCode})  ${areaCode} - ${lineNumber} `;
   }
+  if (match && phoneNumberString?.length <= 6) {
```
Comment (AhmadAlShwaiki) [inline src/utils/helpers.js:72]:
```suggestion
  else {
```

Comment (IbrahimAldereni-socuimtech) [reply src/utils/helpers.js:72]: prettier remove the else statement, and else if statement

Comment (IbrahimAldereni-socuimtech) [reply src/utils/helpers.js:72]: and here iam checking two conditions, if i put else statement it will be executed even if the regex don't match

**Review by AhmadAlShwaiki — APPROVED** [review]

**Review by AhmadAlShwaiki — APPROVED** [review]

**Pull request merged**